applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_geometry_utilities.cpp

```diff
@@ -0,0 +1,259 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_geometries/line_interface_geometry.h"
+#include "custom_utilities/geometry_utilities.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(GeometryUtilities_RotationMatrixForHorizontalInterfaceIsIdentityMatrix,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    PointerVector<Node> nodes;
+    nodes.push_back(Kratos::make_intrusive<Node>(1, 0.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(2, 5.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(3, 0.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(4, 5.0, 0.0, 0.0));
+    const LineInterfaceGeometry<Line2D2<Node>> geometry(1, nodes);
+
+    // Note that only the first component of the local coordinate is used
+    const auto local_coordinate = array_1d<double, 3>{0.0, 0.0, 0.0};
+    Matrix     rotation_matrix =
```

Comment:
`const auto` rather than repeating `Matrix`?
```suggestion
    const auto rotation_matrix =
```

The same suggestion can be applied to the other test cases.

Reply:
Done, was a bit too careful here with auto, Matrix and KRATOS_EXPECT_MATRIX_NEAR